test/acceptance/js/CRUDTests.js

```diff
@@ -0,0 +1,335 @@
+/* eslint-disable
+    handle-callback-err,
+    no-return-assign,
+*/
+// TODO: This file was created by bulk-decaffeinate.
+// Fix any style issues and re-enable lint.
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+const {expect} = require('chai');
+const mongojs = require("mongojs");
+const {ObjectId} = mongojs;
+
+const Settings = require('settings-sharelatex');
+const db = mongojs(Settings.mongo.url, ['tags']);
+
+const TagsApp = require('./helpers/TagsApp');
+const TagsClient = require('./helpers/TagsClient');
+
+describe('CRUDTests', function() {
+	before(function(done) { return db.tags.createIndex({
```

Comment:
I notice that we use tabs for indent in these test files. Could we change this to spaces? Or even better, will `prettier` change it for us?

Reply:
Prettier is configured (by default) to indent using spaces. Somehow this file never got hit by eslint. I've put together a pr to fix this https://github.com/overleaf/dev-environment/pull/308. Once it is merged I will run it again and force push.